src/components/chip-set/chip-set.tsx

```diff
@@ -83,10 +83,10 @@ export class ChipSet {
     public searchLabel: string;
 
     /**
-     * Label to display as `title` and `aria-label` for button that clears all chips
+     * ***DEPRECATED*** Label to display as `title` and `aria-label` for button that clears all chips
```

Comment:
```suggestion
     * Label to display as `title` and `aria-label` for button that clears all chips
     * @deprecated
```

Reply:
Resolving this as I removed the property.